test/integration/mock/repo-data-api/data/repos.js

```diff
@@ -272,6 +272,54 @@ module.exports = [
 
 		// mock use only
 		_versions: ['1.5.0', '1.4.0', '1.3.0', '1.2.0', '1.1.0', '1.0.0']
-	}
+	},
+
+	// v3 Origami component
+	{
+		name: 'o3-example-active',
```

Comment:
Would the presence of this help us with our tests in `origami-build-service`, i.e this will give the build service a component to work with.